### PR TITLE
OC-5923 remove report bug url, update supportURL

### PR DESCRIPTION
**Pull request by rooney**

### CODE REVIEW

File: core/src/main/java/org/akaza/openclinica/dao/core/CoreResources.java [hunk 1]
```diff
@@ -363,7 +363,7 @@ private Properties setDataInfoProperties() {
 
         String supportURL = DATAINFO.getProperty("supportURL");
         if (supportURL == null || supportURL.isEmpty())
-            supportURL = "http://www.openclinica.org/OpenClinica/3.1/support/";
+            supportURL = "https://www.openclinica.com/support/";
```
Comment (kkrumlian) [inline core/src/main/java/org/akaza/openclinica/dao/core/CoreResources.java]: Please make this support URL : https://www.openclinica.com/support

Comment (rooney) [inline core/src/main/java/org/akaza/openclinica/dao/core/CoreResources.java]: @kkrumlian but it's already that? (in datainfo.properties)
Or do you mean that I should remove the trailing slash?

Comment (kkrumlian) [inline core/src/main/java/org/akaza/openclinica/dao/core/CoreResources.java]: Yes please.

**Comment by kkrumlian** [comment]
:+1: Looks good

**Pull request merged**